CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ scripts/setup-cross.sh
 
 ## Project structure and architecture
 
-For a detailed overview of the Runwasi structure and architecture, please refer to the [Architecture Documentation](./docs/src/user-guide/architecture.md).
+For a detailed overview of the Runwasi structure and architecture, please refer to the [Architecture Documentation](https://runwasi.dev/developer/architecture.html).
```

Comment:
Does this check happen before or after deploy? If it happens before deploy, then new links added with new files will not pass the link check b/c the file would not be published to runwasi.dev yet.

Reply:
The linkChecker runs every midnight and it doesn't run for a deployment. So it might be before or after. Is your concern that there might be a period where we checked in broken links and not got caught by the CI in a timely manner? I think as long as we keep an eye on the CI and fix the broken links timely I am fine with that.

Reply:
Interesting. I would imagine a nightly checker would be best for checking links that have broken leading to sites that are not controlled by the project. For links that are controlled by the project, they can be verified at PR / deployment time.

Reply:
Moved the linkChecker to the ci